docs/assets/css/palette.css

```diff
@@ -0,0 +1,12 @@
+:root > * {
+  --srai-brand-color: #01fdb1;
```

Comment:
TBH, I feel a lot of eyestrain looking at this color in the docs. Maybe a little bit darker color like `#00eda5`?